lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -45,82 +41,79 @@ abstract class SqlParserTestBase : TestBase() {
     protected fun parse(source: String): ExprNode = parser.parseExprNode(source)
     protected fun parseToAst(source: String): PartiqlAst.Statement = parser.parseAstStatement(source)
 
+    /**
+     * This method is used by test cases, to test with PIG AST, while the expected PIG AST is a PIG builder
```

Comment:
> This method is used by test cases...

This kdoc isn't overly descriptive. I would add some detail about what's actually being tested in these these functions. Even just summarizing all the parsing and (de)serialization checks would be helpful.

Reply:
Done in next revision.